documentation/operators/src/sandbox.md

```diff
@@ -45,6 +51,6 @@ export NYMNODE_CONFIG_ENV_FILE_ARG=<PATH/TO/sandbox.env>
 
 ## Sandbox Token Faucet
 
-To run your nodes in the sandbox environment, you need testnet version of NYM token, that can be aquired at [faucet.nymtech.net](https://faucet.nymtech.net).
+To run your nodes in Sandbox environment, you need testnet version of NYM token, that can be aquired from [faucet.nymtech.net](https://faucet.nymtech.net).
```

Comment:
We should pick on whether to always capitalise `Sandbox` or not - don't mind either way, just for consistency.

Reply:
I did before this note - so let ***S***andbox be it!